README.md

```diff
@@ -346,6 +346,21 @@ To stop sharing existing factors created with **App Groups**, uncheck the **App
 
 > This will restrict access to the factors and will not affect the main application in which the data was initially created.
 
+---
+
+## Allow Factors migration
```

Comment:
Please update the readme with the new way to pass the value (create factor payload)